app/views/admin/project_steps/_form.html.slim

```diff
@@ -13,32 +18,32 @@ div.panel-body class=('new-record' if step.new_record?)
 
     = f.error_notification
     = f.translatable_inputs do |l|
-      = f.input "summary_#{l}".to_sym, wrapper: :step_form_left_12,
+      = f.input "summary_#{l}".to_sym,
         input_html: {data: {translatable: 'common.summary'}},
-        placeholder: I18n.t('common.summary', locale: l)
-      = f.input "details_#{l}".to_sym, as: :text, wrapper: :step_form_left_12,
+        placeholder: I18n.t('common.summary', locale: l),
```

Comment:
Why is the data-translatable value `common.summary`? Shouldn't it just be `summary`. Not sure when this happened but can you fix?